### DISCUSSION TITLE
Как получить значение из последней строки?

### DISCUSSION THREAD

**Discussion by UPRLS9000** [totumonline/totum-ru-issues-and-discussions#456] [Как это сделать?] [answered]
Здравствуйте. Как получить значение из последней строки колонки?
Вроде видел в одном из видео в учебном курсе, но теперь не могу найти. подскажите, пожалуйста.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Select с параметром order и направлением desc

**Comment by UPRLS9000** [comment]
Спасибо
